components/Video/Video.tsx

```diff
@@ -0,0 +1,26 @@
+const Video = ({ onClose }: { onClose: () => void }) => {
+  return (
+    <div
+      className="absolute left-0 top-0 w-full h-full z-[15]
+      flex items-center justify-center"
+      onClick={(e: any) => {
+        if (e.target === e.currentTarget) onClose()
```

Comment:
The popup with the YouTube iframe only appears when isPopupOpen is true.
if (e.target === e.currentTarget) {
              setPopupOpen(false);
              onClose();
            }